### ISSUE TITLE
Add `contains` condition processing

### ISSUE THREAD

**Issue by klimov-paul** [yii2tech/filedb#5]
Some fields may conatain array as data, for example:

```php
[
    1 => [
        'name' => 'foo',
        'categoryIds' => [
            1,
            2,
            3,
        ],
    ],
];
```

The should be ability to query rows, which holds some value inside array-type attribute.

Keys `contains` and `not contains` can be used for such kind of conditions.

**Comment by klimov-paul** [comment]
There is no need of introduction of the extra condition key - `IN` should be used.

**Comment by klimov-paul** [comment]
Resolved by commit d603ca2ee891f1cc37cae218c469bbf49709d08b
